mypy/nodes.py

```diff
@@ -1007,9 +1007,12 @@ def accept(self, visitor: NodeVisitor[T]) -> T:
 
 # How mypy uses StrExpr, BytesExpr, and UnicodeExpr:
 # In Python 2 mode:
-# b'x', 'x' -> StrExpr
+# b'x' -> BytesExpr  [new!]
+# 'x' -> StrExpr
 # u'x' -> UnicodeExpr
-# BytesExpr is unused
+# However after `from __future__ import unicode_literals` [also new!]:
```

Comment:
Previously, with `unicode_literals` mypy would assume `b''` was a unicode literal too! (Incorrectly, of course.)